core/cog-shell.c

```diff
@@ -277,13 +285,21 @@ cog_shell_class_init (CogShellClass *klass)
                              G_PARAM_READABLE |
                              G_PARAM_STATIC_STRINGS);
 
+    s_properties[PROP_IGNORE_TLS_ERRORS] =
+        g_param_spec_boolean ("ignore-tls-errors",
+                              "Ignore TLS errors",
+                              "Wether to ignore TLS errors or not",
+                              FALSE,
+                              G_PARAM_READWRITE);
+
     g_object_class_install_properties (object_class, N_PROPERTIES, s_properties);
 }
 
 
 static void
-cog_shell_init (CogShell *shell G_GNUC_UNUSED)
+cog_shell_init (CogShell *shell)
 {
+    PRIV (shell)->ignore_tls_errors = FALSE;
```

Comment:
This line is not really needed because the object memory region is zeroed when it gets allocated; therefore the value is already `FALSE` due to that.